cargo-insta/Cargo.toml

```diff
@@ -11,21 +11,24 @@ keywords = ["snapshot", "testing", "jest"]
 categories = ["development-tools::cargo-plugins"]
 edition = "2018"
 readme = "README.md"
-rust-version = "1.61.0"
+rust-version = "1.64.0"
```

Comment:
Note this bumps `cargo-insta` to 1.64, which was released ~2 years ago, Sep 2022

`insta` remains on `1.60.0`